src/core/exec-invoke.c

```diff
@@ -2251,6 +2251,44 @@ static int setup_private_users(PrivateUsers private_users, uid_t ouid, gid_t ogi
         return 1;
 }
 
+static int setup_private_pids(const ExecContext *c, ExecParameters *p) {
+        _cleanup_(pidref_done) PidRef pidref = PIDREF_NULL;
+        int r, q;
+
```

Comment:
we typically validate function parameters with assert() for programming mistakes, i.e. add here:

```
assert(c);
assert(p);
```